example/ios/Runner/GeneratedPluginRegistrant.m

```diff
@@ -3,12 +3,10 @@
 //
 
 #import "GeneratedPluginRegistrant.h"
-#import <flutter_auth0/FlutterAuth0Plugin.h>
 
 @implementation GeneratedPluginRegistrant
 
 + (void)registerWithRegistry:(NSObject<FlutterPluginRegistry>*)registry {
-  [FlutterAuth0Plugin registerWithRegistrar:[registry registrarForPlugin:@"FlutterAuth0Plugin"]];
```

Comment:
Now it doesn't work on iOS anymore

Reply:
That edit was made automatically when upgrading, I wasn't sure about it, I'll rollback the iOS changes